perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -654,7 +654,7 @@ sub response_templates {
 #     Note:   this only makes sense when called on a problem that has been sent!
 sub can_display_external_id {
     my $self = shift;
-    if ($self->external_id && $self->send_method_used && $self->bodies_str =~ /2237/) {
+    if ($self->external_id && $self->send_method_used && $self->bodies_str =~ /(2237|2550)/) {
```

Comment:
It's tricky, isn't it, because the cobrand `$c->cobrand` is referring to might not be the `Cobrand` class for the body that the report was sent to. Is there a method for getting the correct `Cobrand` based on the problem's `bodies_str`?